tck/src/main/java/org/eclipse/microprofile/config/tck/EmptyValue/EmptyValuesTest.java

```diff
@@ -55,25 +55,7 @@ public static Archive deployment() {
     @Inject
     private EmptyValuesBean emptyValuesBean;
 
-    @Inject
-    private Config config;
-
-    @Test
-    public void testEmptyStringValues() {
-        assertEquals(emptyValuesBean.getStringValue(), "");
-    }
-
-    @Test
-    public void testEmptyStringProgrammaticLookup() {
-        System.setProperty(EMPTY_PROPERTY, "");
-        String stringValue = config.getValue(EMPTY_PROPERTY, String.class);
-        assertEquals(stringValue, "");
-        System.clearProperty(EMPTY_PROPERTY);
-    }
-
     @Test
-    public void testEmptyStringPropertyFromConfigFile() {
-        String stringValue = config.getValue(PROP_FILE_EMPTY_PROPERTY, String.class);
-        assertEquals(stringValue, "");
+    public void test() {
```

Comment:
Is there nothing to test?

Reply:
This is just a negative test case and the deployment will fail. The test there is for reporting true. This is the pattern we use for deployment failures (e.g. [this](https://github.com/eclipse/microprofile-config/blob/master/tck/src/main/java/org/eclipse/microprofile/config/tck/broken/MissingConverterOnInstanceInjectionTest.java) one)